### PR TITLE
Feature/32996 add download button for image attachment

### PR DESCRIPTION
**Pull request by sushmi21**
Add download button for image attachment. 

- Run the new test 'downloadableImage' and check if the download button is shown below the image.

### CODE REVIEW

File: cypress/fixtures/messages/downloadableImage.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "data": {
+        "_cognigy": {
+            "_default": {
+                "_image": {
+                    "type": "image",
+                    "imageUrl": "https://placekitten.com/300/300",
+                    "buttons": [
+                        {
+                            "id": 0.17194659386513078,
+                            "payload": "",
```
Comment (dshire) [inline cypress/fixtures/messages/downloadableImage.json:12]: the id and payload fields are not used or are they? I didn't see them in your other PR. are they needed for the test?

File: src/plugins/messenger/MessengerPreview/components/MessengerMediaTemplate/MessengerMediaTemplate.tsx [hunk 2]
```diff
@@ -67,8 +80,26 @@ export const getMessengerMediaTemplate = ({
             }, 100);
         }
 
-        const { media_type, url, altText } = element as IFBMMediaTemplateUrlElement;
-        // TODO add buttons
+        const imageButtons = () => {
```
Comment (dshire) [inline src/plugins/messenger/MessengerPreview/components/MessengerMediaTemplate/MessengerMediaTemplate.tsx:83]: should we not return null here if buttons is falsy? else you would return an empty div

File: cypress/fixtures/messages/downloadableImage.json (hunk 1 above)
Comment (sushmi21) [reply cypress/fixtures/messages/downloadableImage.json:12]: Yes, they were added for tests. I first wrote a test that uses the id. But I will remove this as it is not used anymore

Comment (sushmi21) [reply cypress/fixtures/messages/downloadableImage.json]: removed

**Review by dshire — APPROVED** [review]

**Pull request merged**